src/composable/ConditionalOrder.spec.ts

```diff
@@ -153,6 +163,72 @@ describe('cabinet', () => {
     expect(mockCabinet.mock.calls).toHaveLength(1)
 
     // THEN: we expect the params to be the owner and 0x0
-    expect(mockCabinet.mock.calls[0]).toEqual([owner, constants.HashZero])
+    expect(mockCabinet.mock.calls[0]).toEqual([OWNER, constants.HashZero])
+  })
+})
+
+describe('Poll Single Orders', () => {
+  const mockSingleOrders = jest.fn()
+  const mockGetTradeableOrderWithSignature = jest.fn()
+  const param = { owner: OWNER, chainId: 1, provider: {} } as PollParams
+
+  const signature = '0xd8dA6BF26964aF9D7eEd9e03E53415D37aA96045'
+  beforeEach(() => {
+    jest.resetAllMocks()
+
+    mockGetComposableCow.mockReturnValue({
+      callStatic: {
+        singleOrders: mockSingleOrders,
+      },
+      getTradeableOrderWithSignature: mockGetTradeableOrderWithSignature,
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    } as any)
+  })
+
+  test.only('[SUCCESS] Happy path', async () => {
+    // GIVEN: An order that is authorised
+    mockSingleOrders.mockReturnValue(true)
+
+    // GIVEN: And a getTradeableOrderWithSignature that doesn't revert
+    mockGetTradeableOrderWithSignature.mockReturnValue([DISCRETE_ORDER, signature])
+
+    // When we poll
+    const pollResult = await SINGLE_ORDER.poll(param)
+
+    // THEN: we expect a CALL to getTradeableOrderWithSignature with the owner, params, off-chain input, and no-proof
+    expect(mockGetTradeableOrderWithSignature.mock.calls).toHaveLength(1)
+    expect(mockGetTradeableOrderWithSignature.mock.calls[0]).toEqual([
+      OWNER,
+      SINGLE_ORDER.leaf,
+      SINGLE_ORDER.offChainInput,
+      [],
+    ])
+
+    // THEN: We expect a SUCCESS result, which returns the order and the signature
+    pollResult.result === PollResultCode.UNEXPECTED_ERROR && console.error(pollResult.error)
```

Comment:
Hrmm, I don't properly understand what this line does?

Reply:
Translating it:

```ts
if (pollResult.result === PollResultCode.UNEXPECTED_ERROR) {
  console.error(pollResult.error)
}
```

Agree, not very legible.

Reply:
this line is not supoused to be there. I forgot to delete it. 

It just prints the error. This is how I debugged the test until i made it work. 
The guard condition is required in order to access the error prop, since this prop is only present for `UNEXPECTED_ERROR`s 

I will delete